javascript/ql/test/library-tests/TaintTracking/sanitizer-guards.js

```diff
@@ -63,3 +63,14 @@ function phi() {
   }
   sink(x); // OK
 }
+
+function phi2() {
+  let x = source();
+
+  if (something(x) || isSafe(x)) {
+    // this input to the phi node for 'x' is not fully sanitized
+  } else {
+    x = null;
+  }
+  sink(x); // OK
```

Comment:
This should be `// OK, but still flagged` or something like that.

Reply:
Copy pasta. Changed to `// NOT OK`.